### PR TITLE
[Ishan] - Sage - basics of blockchain

### PR DESCRIPTION
**Pull request by Ishanpathak1**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

**Comment by developerkunal** [comment]
@Ishanpathak1 your image url in users.tsx is wrong fix that

**Comment by developerkunal** [comment]
Looks good to me

**Comment by developerkunal** [comment]
@viral-sangani can you check and approve this

File: blog/02-03-2023-basics-of-blockchain/basics-of-blockchain.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+Static NFT- Is a digital asset that won’t be able to change or modified in any way, Once it is created you won’t be able to change that and this makes it unique
+
+Dynamic NFT-Whereas in a Dynamic NFT can be modified in the future, such as a game or an environment.
+
+Static NFTs are more common to see around.
+
+After reading these portions if you want to be the owner of a cool NFT, you can check that out on these amazing marketplaces- OpenSea, Rarible, SuperRare, and KnownOrigin
+
+2. DAO
```
Comment (viral-sangani) [inline blog/02-03-2023-basics-of-blockchain/basics-of-blockchain.md:115]: Please add extra line after here.
Note, there are lot of changes that requires adding new line, so I suggest using Prettier for this. That will format the markdown properly.

Thanks

File: blog/02-03-2023-basics-of-blockchain/basics-of-blockchain.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
+---
+title: Blockchain Basics- An Introduction to Web3 Terms and Concepts with Celo
+description: In this article we will be leanring about the basics of the blockchain and the technologies around it such as NFT, DAO, DEFI and also about the Web3 with Celo.
```
Comment (viral-sangani) [inline blog/02-03-2023-basics-of-blockchain/basics-of-blockchain.md:3]: @Ishanpathak1 One small change, can you decrease the description size to aroung 18/20 works. Larger description sized make the cards look weired in the website.

please make the same changes in users.txs

**Review by viral-sangani — CHANGES_REQUESTED** [review]

File: blog/02-03-2023-basics-of-blockchain/basics-of-blockchain.md (hunk 2 above)
Comment (Ishanpathak1) [reply blog/02-03-2023-basics-of-blockchain/basics-of-blockchain.md]: I have made the changes

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**